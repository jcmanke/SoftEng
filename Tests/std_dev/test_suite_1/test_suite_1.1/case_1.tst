10
5
7
2
1
8
4
3
9
6
