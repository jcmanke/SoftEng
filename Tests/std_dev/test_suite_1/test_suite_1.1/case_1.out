The average score is: 5.5
The Standard Deviation is: 3.02765
